sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -98,7 +98,9 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
 
   /** List of test cases to ignore, in lower cases. */
   private val blackList = Set(
-    "blacklist.sql"  // Do NOT remove this one. It is here to test the blacklist functionality.
+    "blacklist.sql",  // Do NOT remove this one. It is here to test the blacklist functionality.
+    ".ds_store"       // A meta-file that may be created on Mac by Finder App.
```

Comment:
@srowen I did think about it. If we want to keep the mixed case here in this list, then we have to change the code later to lowercase both the sides and then compare ? Would you prefer to change the comparison to the following ?
```
if (blackList.exists(t => testCase.name.toLowerCase.contains(t.toLowerCase))) {
...
}
```